### DISCUSSION TITLE
Support for Jersey annotations and types

### DISCUSSION THREAD

**Discussion by vbazhmin** [smallrye/smallrye-open-api#1645] [Q&A] [answered]
Hello, folks !

Are there any internal discussions towards supporting generation of OpenAPI specification from Jersey annotations/types ? I've stumbled upon situation in which I have legacy code based partially on standard JaxRS and partially on "internal things" of glassfish/jersey like `@FormDataParam`, `FormDataContentDisposition`, `FormDataBodyPart`, etc ....

It could be a nice feature ;)

Regards,
Vladimir

**Comment by phillip-kruger** [comment, marked as answer]
Hi @vbazhmin This is not on our radar, but you are welcome to create a PR with this. Another option would be to change smallrye to allow users to define annotations to scan/map to existing scanning. @MikeEdgar w.d.y.t ?

**Reply by MikeEdgar** [reply]
Yes, it seems like it could be another extension like JAX-RS or Spring, but it could perhaps extend from the JAX-RS one if some annotations are shared.
